test/common/halide_test_dirs.h

```diff
@@ -16,6 +16,46 @@
 namespace Halide {
 namespace Internal {
 
+std::string get_env(const char *name) {
+#ifdef _MSC_VER
+    char buf[MAX_PATH];
+    size_t read = 0;
+    getenv_s(&read, buf, name);
```

Comment:
The ignore-the-result pattern is used elsewhere in Halide. I'll update all of them.